.changelogs/llms-rest-searchable-posts.yml

```
@@ -0,0 +1,3 @@
+significance: minor
+type: added
+entry: Post based resources like couress, memberships, section, lessons... are now searchable 
```

Entries should end in full stops... I kind of have a CS rule for this but it doesn't run automatically yet :-p

```suggestion
entry: Post based resources like couress, memberships, section, lessons... are now searchable.
```